app.js

```diff
@@ -52,3 +52,23 @@ function fakeFind(array, callback) {
   }
   return undefined;
 }
+////forEach
+
+
+function fakeForEach(array, callback) {   
```

Comment:
Fijate un par de cosas: 
1. El for se puede reemplazar por un for of.
2. El if (y el return true) no son necesarios. Sólo tenés que ejecutar la función para cada uno de los elementos.